ui/src/app/views/workspace-about.tsx

```diff
@@ -2,151 +2,276 @@ import {Component} from '@angular/core';
+  render() {
+    const {profileState: {profile}} = this.props;
+    const {cdrVersion, workspace, workspaceUserRoles, googleBucketModalOpen, sharing} = this.state;
+    return <div style={styles.mainPage}>
+      <ResearchPurpose data-test-id='researchPurpose'/>
+      <div style={styles.rightSidebar}>
+        <div style={styles.shareHeader}>
+          <h3 style={{marginTop: 0}}>Collaborators:</h3>
+          <TooltipTrigger content={ShareTooltipText()}>
+            <InfoIcon style={{margin: '0 0.3rem'}}/>
+          </TooltipTrigger>
+          <Button style={{height: '22px', fontSize: 12, marginRight: '0.5rem',
+            maxWidth: '13px'}} disabled={workspaceUserRoles.length === 0}
+                  data-test-id='workspaceShareButton'
+                  onClick={() => this.setState({sharing: true})}>Share</Button>
         </div>
-        <div style={styles.sectionText}>{ResearchPurposeDescription}</div>
-        <div style={styles.sectionHeader}>Primary purpose of the project</div>
-        <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
-                           contentLeft={selectedResearchPurposeItems.slice(0, rpItemsHalfLen)}
-                           contentRight={selectedResearchPurposeItems.slice(rpItemsHalfLen)}/>
-        <div style={styles.sectionHeader}>
-          Reason for choosing <i>All of Us</i> data for your investigation</div>
-        <div style={styles.sectionText}>{workspace.researchPurpose.reasonForAllOfUs}</div>
-        <div style={styles.sectionHeader}>Area of intended study</div>
-        <div style={styles.sectionText}>{workspace.researchPurpose.intendedStudy}</div>
-        <div style={styles.sectionHeader}>Anticipated findings from this study</div>
-        <div style={styles.sectionText}>{workspace.researchPurpose.anticipatedFindings}</div>
-        {workspace.researchPurpose.population && <div style={{marginBottom: '1rem'}}>
-          <div style={styles.sectionHeader}>Population area(s) of focus</div>
-          <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
-                             contentLeft={this.getSelectedPopulationsSlice(true)}
-                             contentRight={this.getSelectedPopulationsSlice(false)}/>
+        {workspaceUserRoles.length > 0 ?
+          <React.Fragment>
+            {workspaceUserRoles.map((user, i) =>
+              <div key={i} data-test-id={'workspaceUser-' + i}>
+                {user.role + ' : ' + user.email}
+              </div>)}
+          </React.Fragment> :
+          <Spinner size={50} style={{display: 'flex', alignSelf: 'center'}}/>}
+        <div>
+          <h3 style={{marginBottom: '0.5rem'}}>Workspace Information:
+            <TooltipTrigger content={WorkspaceInfoTooltipText()}>
+              <InfoIcon style={{margin: '0 0.3rem'}}/>
+            </TooltipTrigger>
+          </h3>
+          <div style={styles.infoBox} data-test-id='cdrVersion'>
+            <div style={styles.infoBoxHeader}>Dataset</div>
```

Comment:
Not sure whether this is an actionable PR comment, but now that we have data sets, this language makes me feel weird... might be worth checking with Lou

Reply:
i'll check with lou, gonna leave it for this PR though bc it is not a regression